src/session10/homework/animal/Animal.java

```diff
@@ -0,0 +1,14 @@
+package session10.homework.animal;
```

Comment:
Please use the name of the packages to point out what kind of resources share. For example, if we have multiple inheritance problems in each package, we can describe the type of inheritance used in that package.